### PR TITLE
[MRG+2] Fix repo links

### PR DESCRIPTION
**Pull request by aaronreidsmith**
# Description

This PR fixes some repo links in our comments and docs. I know they would automatically redirect from `https://www.github.com/tgsmith61591/pmdarima` to `https://www.github.com/alkaline-ml/pmdarima`, but it is better to have them point to the correct URL from the get go.

## Type of change

Please delete options that are not relevant.

- [X] Documentation change

# How Has This Been Tested?

N/A

# Checklist:

N/A

### CODE REVIEW

File: examples/quick_start_example.ipynb [hunk 1]
```diff
@@ -6,7 +6,7 @@
...
+    "Pyramid bring R's [`auto.arima`](https://www.rdocumentation.org/packages/forecast/versions/7.3/topics/auto.arima) functionality to Python by wrapping statsmodel [`ARIMA`](https://github.com/statsmodels/statsmodels/blob/master/statsmodels/tsa/arima_model.py) and [`SARIMAX`](https://github.com/statsmodels/statsmodels/blob/master/statsmodels/tsa/statespace/sarimax.py) models into a singular scikit-learn-esque estimator ([`pmdarima.arima.ARIMA`](https://github.com/alkaline-ml/pmdarima/blob/master/pmdarima/arima/arima.py)) and adding several layers of degree and seasonal differencing tests to identify the optimal model parameters.\n",
```
Comment (tgsmith61591) [inline examples/quick_start_example.ipynb]: Do you mind changing this to pmdarima?

File: examples/quick_start_example.ipynb [hunk 2]
```diff
@@ -6,7 +6,7 @@
...
     "\n",
     "__Pyramid ARIMA models:__\n",
```
Comment (tgsmith61591) [inline examples/quick_start_example.ipynb]: As well as this.

File: examples/quick_start_example.ipynb (hunk 1 above)
Comment (aaronreidsmith) [reply examples/quick_start_example.ipynb]: Fixed!

File: examples/quick_start_example.ipynb (hunk 2 above)
Comment (aaronreidsmith) [reply examples/quick_start_example.ipynb]: Fixed!

**Pull request merged**